snippets/pod-volume-restore-snapshot-read-only.adoc

```diff
@@ -0,0 +1,20 @@
+// Text snippet included in the following modules:
+//
+// * openshift-docs/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+// * openshift-docs/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+.PodVolumeRestore fails with the error `.../.snapshot: read-only file system`
+====
+The `.../.snapshot` directory is a snapshot copy directory, which is used by several NFS servers. This directory has read-only access by default, so velero cannot restore to this directory.
+
+You should not give write access to this `.snapshot` directory to velero, and you should disable client access to this directory.
```

Comment:
```suggestion
Do not give Velero write access to this `.snapshot` directory, and disable client access to this directory.
```
From IBM Style: "Whenever possible, rewrite _should_ in a more direct way. Do not use _should_ to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”."